index.html

```diff
@@ -64,7 +61,186 @@ <h4>LET'S CONNECT</h4>
+            <li>CANOPY</li>
+            <img src="./image/point.png" />
+            <li>Back End Dev</li>
+            <img src="./image/point.png" />
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="work-button">See Project</button>
+        </div>
+      </section>
+      <section id="section3">
+        <h2>
+          About
+          Myself
+        </h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <h3>LET'S CONNECT</h4>
+        <div id="icons">
+          <ul id="list-icons">
+            <li class="img">
+              <a href="#">
+                <img src="./image/tweeter.png" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/Linkedin icon.jpg" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/m.png" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/github.png" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/hand.png" alt="" />
+              </a>
+            </li>
+          </ul>
+        </div>
+        <button class="section3-button">Get my resume</button>
+        <div class="menu-categories">
+          <ul class="categories">
+            <li class="li-categories">Languages</li>
+            <img class="li-img" src="./image/arrow-down.png" alt="" />
+            <ul class="dropdown-menu">
+              <li>
+                <img src="./image/Ellipse1.png" />
+                JavaScript
+              </li>
+              <li>
+                <img src="./image/Ellipse2.png" />
+                HTML
+              </li>
+              <li>
+                <img src="./image/Ellipse3.png" />
+                CSS
+              </li>
+            </ul>
+          </ul>
+          <ul class="categories">
+            <li class="li-categories">Framekors
+            </li>
+            <img class="li-img" src="./image/Union.png"/>  
+          </ul>
+          <ul class="categories">
+            <li class="li-categories">Skills</li>
+            <img class="li-img" src="./image/Union.png"/>
+          </ul>
+        </div>
+      </section>
+      <section class="section4">
+          <h2>Contact me</h2>

```

Comment:
Please consider adding the same margin-top to the section as used in the Figma design to meet the project requirements.

```
    margin-top: 176.85px;
```